src/Skjema/Utdanning.elm

```diff
@@ -0,0 +1,116 @@
+module Skjema.Utdanning exposing (Felt(..), UtdanningSkjema(..), beskrivelse, encode, fradato, init, navarende, nuskode, oppdaterFelt, studiested, tildato, utdanningsretning)
+
+import Cv.Utdanning exposing (Utdanning, Yrkesskole(..))
+import Json.Encode
+
+
+type UtdanningSkjema
+    = UtdanningSkjema UtdanningSkjemaInfo
+
+
+type alias UtdanningSkjemaInfo =
+    { studiested : String
+    , utdanningsretning : String
+    , fradato : String
+    , tildato : String
+    , beskrivelse : String
+    , navarende : Bool --- bool? ref Dtoen
+    , nuskode : String
+    }
+
+
+type Felt
+    = Studiested
+    | Utdanningsretning
+    | Beskrivelse
+    | Fradato
+    | Tildato
+    | Navarende
+    | Nuskode
+
+
+studiested : UtdanningSkjema -> String
+studiested (UtdanningSkjema info) =
+    info.studiested
+
+
+utdanningsretning : UtdanningSkjema -> String
+utdanningsretning (UtdanningSkjema info) =
+    info.utdanningsretning
+
+
+fradato : UtdanningSkjema -> String
+fradato (UtdanningSkjema info) =
+    info.fradato
+
+
+tildato : UtdanningSkjema -> String
+tildato (UtdanningSkjema info) =
+    info.tildato
+
+
+beskrivelse : UtdanningSkjema -> String
+beskrivelse (UtdanningSkjema info) =
+    info.beskrivelse
+
+
+navarende : UtdanningSkjema -> Bool
+navarende (UtdanningSkjema info) =
+    info.navarende
+
+
+nuskode : UtdanningSkjema -> String
+nuskode (UtdanningSkjema info) =
```

Comment:
Syns ikke dere burde ha noe felt for nuskode i det hele tatt. For dette er egentlig "nivå", så dere kan heller ha et felt for `nivå` og så kan dere konvertere til nuskode i encode.